results_paper/compare_metrics.ipynb

```diff
@@ -0,0 +1,399 @@
+{
+ "cells": [
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "import numpy as np\n",
+    "import pandas as pd\n",
+    "import pickle\n",
+    "\n",
+    "import matplotlib as mpl\n",
+    "# print(mpl.rcParams.items)\n",
+    "mpl.use('Agg')\n",
+    "mpl.rcParams['text.usetex'] = False\n",
+    "mpl.rcParams['mathtext.rm'] = 'serif'\n",
+    "mpl.rcParams['font.family'] = 'serif'\n",
+    "mpl.rcParams['font.serif'] = ['Times New Roman']\n",
+    "# mpl.rcParams['font.family'] = ['Times New Roman']\n",
+    "mpl.rcParams['axes.titlesize'] = 25\n",
+    "mpl.rcParams['axes.labelsize'] = 20\n",
+    "mpl.rcParams['xtick.labelsize'] = 15\n",
+    "mpl.rcParams['ytick.labelsize'] = 15\n",
+    "mpl.rcParams['savefig.dpi'] = 250\n",
+    "mpl.rcParams['figure.dpi'] = 250\n",
+    "mpl.rcParams['savefig.format'] = 'pdf'\n",
+    "mpl.rcParams['savefig.bbox'] = 'tight'\n",
+    "import matplotlib.pyplot as plt\n",
+    "%matplotlib inline"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "def truncate_colormap(cmap, minval=0.0, maxval=1.0, n=100):\n",
+    "    new_cmap = mpl.colors.LinearSegmentedColormap.from_list(\n",
+    "        'trunc({n},{a:.2f},{b:.2f})'.format(n=cmap.name, a=minval, b=maxval),\n",
+    "        cmap(np.linspace(minval, maxval, n)))\n",
+    "    return new_cmap\n",
+    "\n",
+    "cmap = plt.get_cmap('hot_r')\n",
+    "fave_cmap = truncate_colormap(cmap, 0.35, 1.0)"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "import os\n",
+    "os.getcwd()"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "# metric_vals = pd.read_csv('results_paper/metrics_vals.txt', delimiter='\\t', skiprows=1, names=('garbage', 'PLAsTiCC-weighted LogLoss', 'Brier score', 'unweighted LogLoss'))\n",
+    "metric_vals = pd.read_csv('metrics_vals.txt', delimiter='\\t', skiprows=1, names=('garbage', 'PLAsTiCC', 'Brier', 'LogLoss')).drop(columns='garbage')"
```

Comment:
I will ping you when I've got the version with class 99, so that the figure has both cases in it.